tests/datatypes/test_bytes.py

```diff
@@ -28,6 +30,12 @@ def test_capitalize(self):
             print(b'2015638687'.capitalize())
         """)
 
+    @unittest.skip("Move this to test_capitalize upon resolution of #530")
```

Comment:
> skip is for when a test may/may not run due to some other condition (e.g., whether a setting has been turned on)

Isn't that the case only for [`skypIf`](https://docs.python.org/3/library/unittest.html#skipping-tests-and-expected-failures) ?

In my understanding, the [argument for skip() is the reason why the test should be skipped](https://docs.python.org/3/library/unittest.html#skipping-tests-and-expected-failures).

I think skip is more appropriate here because it's not a true failure of the test (as in, assertion failing), because the code never gets to run (and therefore we never get to the point of executing the assertion itself).

What do you think?